validator/client/src/main/java/tech/pegasys/teku/validator/client/slashingriskactions/DoppelgangerDetectionShutDown.java

```diff
@@ -11,21 +11,24 @@
  * specific language governing permissions and limitations under the License.
  */
 
-package tech.pegasys.teku.validator.client.doppelganger;
+package tech.pegasys.teku.validator.client.slashingriskactions;
 
 import static tech.pegasys.teku.infrastructure.logging.StatusLogger.STATUS_LOG;
 
 import java.util.List;
 import java.util.stream.Collectors;
 import tech.pegasys.teku.bls.BLSPublicKey;
 
-public class DoppelgangerDetectionShutDown implements DoppelgangerDetectionAction {
+public class DoppelgangerDetectionShutDown implements SlashingRiskDetectionAction {
+
   @Override
-  public void perform(final List<BLSPublicKey> doppelgangers) {
-    STATUS_LOG.exitOnDoppelgangerDetected(
-        doppelgangers.stream()
-            .map(BLSPublicKey::toAbbreviatedString)
-            .collect(Collectors.joining(", ")));
-    System.exit(1);
+  public void perform(final List<BLSPublicKey> pubKeys) {
+    if (!pubKeys.isEmpty()) {
+      STATUS_LOG.exitOnDoppelgangerDetected(
+          pubKeys.stream()
+              .map(BLSPublicKey::toAbbreviatedString)
+              .collect(Collectors.joining(", ")));
+      System.exit(1);
```

Comment:
we probably need exit code 2 here too

Reply:
Yeah that would be more appropriate

Reply:
Updated the documentation to indicate that too in my documentation PR: https://github.com/Consensys/doc.teku/pull/526